doc/federation/federation.rst

```diff
@@ -0,0 +1,87 @@
+.. _federation/federation:
+
+========================
+The BinderHub Federation
+========================
+
+While it may seem like ``mybinder.org`` is a single website, it is
+in fact a **federation** of teams that deploy public BinderHubs to serve
+the community. This page lists the BinderHubs that currently help
+power ``mybinder.org``.
+
+Visiting ``mybinder.org`` will randomly redirect you to one
+of the following BinderHubs. You may also directly access a BinderHub
+by using its respective sub-domain in your URL.
+
+If your organization is interested in becoming part of the BinderHub
+federation, check out :ref:`federation/joining`.
+
+.. _federation/federation-list:
+
+Members of the BinderHub Federation
+===================================
+
+Here is a list of the current members of the BinderHub federation:
+
+.. include:: data-federation.txt
+
+
+.. _federation/joining:
+
+Joining the BinderHub Federation
+================================
+
+The Binder Project is an open community of developers as well as deployers
+of BinderHub. BinderHub is designed to be used by any organization for
+their own purposes. However, many members of the community also run
+the BinderHub deployments at ``mybinder.org`` as a large, free public service
+to the community.
+
+Behind ``mybinder.org`` is a **federation of BinderHubs** - these are
```

Comment:
How about the following:

Behind mybinder.org is a federation of BInderHubs. This means that there are several independent hubs that each serve a fraction of the traffic created by people clicking links pointing to mybinder.org. Anyone (a company, university or individual) is welcome to deploy a BinderHub that forms part of the federation.'

Adding a new hub to the federation requires a mix of two kinds of resources: compute and human power to operate the hub. The two extremes of this mixture are:
* you donate compute power that the mybinder.org team has full control over which means you don't have to be involved in day to day operations, or
* you donate compute power over which the mybinder.org team does not have full control which means you are also responsible for day to day operations.